irony-cdb.el

```diff
@@ -60,9 +54,29 @@ list of cons where the first element is a set of compile options
 and the second element the working directory expected for these
 commands. The compilation database should return an empty list
 for files that it cannot handle."
+  :group 'irony)
```

Comment:
I think the group is a great addition but I'm not sure about moving the documentation.

company-mode has its documentation in `company-backends` I tried to mimic that here.

Right now it's possible to get this documentation either by `describe-variable irony-cdb-compilation-databases` or via  the customize interface. To my knowledge, with this change it is only accessible via `customize-group irony-cdb`.